### PR TITLE
Pass locale to NCP team 👾

### PR DESCRIPTION
**Pull request by spencersablan**
### Description

- 🗺️ _**Added locale in the API call to NCP**_ - You'll see `locale.x=${locale}` as a query parameter when making the API call to NCP to get the hosted buttons details.

### Why are we making these changes? Include references to any related Jira tasks or GitHub Issues
**JIRA:** [DTPPCPSDK-2192](https://paypal.atlassian.net/browse/DTPPCPSDK-2192)

### Dependent Changes (if applicable)

- ⚠️ We need the NCP team to change their URL validation to allow for query params before we can deploy this! Otherwise, there will be a CORS error. 

### Groups who should review (if applicable)

❤️ Thank you!

### CODE REVIEW

**Review by wsbrunson — APPROVED** [review]

File: src/hosted-buttons/utils.js [hunk 1]
```diff
@@ -77,8 +78,11 @@ const getButtonVariable = (variables: ButtonVariables, key: string): string =>
 export const getHostedButtonDetails: HostedButtonDetailsParams = async ({
   hostedButtonId,
 }) => {
+  const { lang, country } = getLocale();
```
Comment (jshawl) [inline src/hosted-buttons/utils.js:81]: I think you can fix the test failure by mocking `getLocale` here: https://github.com/paypal/paypal-checkout-components/blob/88013eaa39a5f54a66f4befffe589c8e121f6945/src/hosted-buttons/utils.test.js#L20-L27

**Review by jshawl — APPROVED** [review]
Looks good! Added one comment about fixing the CI failure :rocket:

File: src/hosted-buttons/utils.js (hunk 1 above)
Comment (spencersablan) [reply src/hosted-buttons/utils.js:81]: Ahhh, thank you so much!

Comment (spencersablan) [reply src/hosted-buttons/utils.js:81]: This did fix it! I had to add it to index.test.ts as well. 🚀

File: src/hosted-buttons/utils.js [hunk 2]
```diff
@@ -77,8 +78,11 @@ const getButtonVariable = (variables: ButtonVariables, key: string): string =>
...
+  const locale = `${lang}_${country}`;
```
Comment (spencersablan) [inline src/hosted-buttons/utils.js:82]: TODO: Change from underscore to hyphen since this is a REST API.

**Pull request merged**